ls1intum/Artemis
# `Development`: Adopted Docker Proxy settings from Production

### Checklist
#### General




### Motivation and Context


Not all Nginx Proxy settings from the Artemis Ansible Collection have yet been integrated in the Artemis Nginx Docker Container due to time constraints.

### Description

This adds the remaining nginx settings and removes the TODO comments.

### Steps for Testing

1. Start Artemis in a Docker Setup with Proxy
2. check that the proxy works

### Review Progress






#### Code Review
#### Manual Tests
